docs/usage.rst

```diff
@@ -367,6 +367,13 @@ deck-chore's behaviour is defined by these environment variables:
 
     The URL of the Docker daemon to connect to.
 
+.. envvar:: STDERR_LEVEL
+
+    default: ``NOTSET``
+
+    Redirect any deck-chores-internal log above level STDERR_LEVEL to STDERR, string or integer required.
+    The default ``NOTSET`` implies that the STDERR is not in use and any logging is done over STDOUT.
```

Comment:
-> *The default ``NOTSET`` implies that all log messages are emitted to `stdout`*.